acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +84,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
```

Comment:
If the `agent['platform']` isn't captured in the `case` statement below, the assertions will be skipped over completely.  I just took this line out because it seemed redundant to match the platforms to the agent both on this line and in the case statement below.  Does that make sense?